CMakeLists.txt

```diff
@@ -777,6 +777,9 @@ option(ENABLE_CONFORMANCE "Enable conformance to OpenCL standard. \
   Enabling this option this does not guarantee conformance (depends on hardware), \
   but CMake will give errors if options that conflict with conformance are used" OFF)
 
+option(ENABLE_UNFINISHED_EXTENSIONS "Advertise extensions in device queries which \
+might work partially, but have known unfinished features." OFF)
+
```

Comment:
This should have a conflict with ENABLE_CONFORMANCE=ON (that is the purpose of ENABLE_CONFORMANCE flag), like e.g. lines 783-785.

In fact i'm not sure this new CMake flag is required; it's already the behavior (at least for CPU drivers) that we just advertise unfinished features when ENABLE_CONFORMANCE=OFF and disable them when it's ON.